doc/build_apps/create_vm.rst

```diff
@@ -1,33 +1,11 @@
 Create Azure SGX VM
 ===================
 
-.. note:: These steps require an `Azure subscription <https://docs.microsoft.com/en-us/azure/billing/billing-create-subscription#create-a-subscription-in-the-azure-portal>`_.
-
-Creation
---------
-
-The install directory contains (under ``getting_started/``) the Azure and Ansible scripts required to create and setup an SGX-enabled development VM in Azure, with all necessary dependencies to build CCF itself and CCF applications.
+:term:`Azure Confidential Compute` (ACC) offers SGX-enabled `DC`_-series VMs, which can be deployed like other Azure SKUs.
```

Comment:
We should at least mention the available regions for that SKU here.

Reply:
I don't really want to, because there are three SKUs (DC, DCv2 and DCv3), with different region availability, and they change quite often. If we find a good page, we'll link to it, but I'm reluctant to try to maintain an up to date list.